src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/TeleporterSystem.java

```diff
@@ -65,7 +70,8 @@ private String setPlayerTeamToTeleporterTeam(EntityRef player, EntityRef telepor
     }
 
     private void handlePlayerTeleport(EntityRef player, String team) {
-        player.send(new CharacterTeleportEvent(LASUtils.getTeleportDestination(team)));
+        Vector3f randomVector = new Vector3f(-2 + random.nextFloat()*4, -2 + random.nextFloat()*4,0);
```

Comment:
Same remarks as above apply.

Reply:
Just wanted to mention that I am considering the range [-2,2] for the random vector even though the actual range should be [-2.5,2.5] as I didn't want players spawning on the edge of the base.